setup.py

```diff
@@ -136,6 +136,15 @@ def run(self):
 
         SDist.run(self)
 
+        # Print warnings at the end because no one will see warnings before all the normal status
+        # output
+        if os.environ.get('_ANSIBLE_SDIST_FROM_MAKEFILE', False) != 1:
+            warnings.warn('When setup.py sdist is run from outside of the Makefile,'
+                          ' the generated tarball may be incomplete.  Use `make snapshot`'
```

Comment:
Maybe "may be missing some of non-runtime-critical artifacts"?

Reply:
Lots of jargon in that wording.  And there's no guarantee that the missing things will be non-runtime-critical in the future either....

Reply:
It's just an example. I just want it to be less scary.